Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -71,6 +71,22 @@ function Write-TaskSpecificTelemetry
     Write-Telemetry "$codeKey" "EB72CB01-A7E5-427B-A8A1-1B31CCAC8A43"
 }
 
+function Get-AzureCmdletsVersion
+{
+    $module = Get-Module AzureRM -ListAvailable
```

Comment:
AzureRM.Profile version is not same the Azure Powershell release version. The azure platform implementation in this task takes into account Azure Powershell release version, not that of AzureRM.